posthog/api/capture.py

```diff
@@ -377,10 +374,6 @@ def get_event(request):
                 # Legacy solution stays in place
                 processed_replay_events = legacy_preprocess_session_recording_events_for_clickhouse(replay_events)
```

Comment:
🤔  we don't need any of this anymore right?

Reply:
Let's go one step at a time.

This removes the code for the v1 recordings ingestion in the plugin server.

Reply:
🤣

Reply:
should probably have opened in draft... trying to slowly figure out why at some point I stop being able to play back